CRAN check error on r-patched-solaris-x86

Version: 2.0.0
Check: whether package can be installed
Result: ERROR
    Installation failed.
Flavor: r-patched-solaris-x86

```
* installing to library ‘/home/ripley/R/Lib32’
* installing *source* package ‘dga’ ...
** package ‘dga’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
/opt/csw/bin/g++ -std=gnu++11 -I"/home/ripley/R/gcc/include" -DNDEBUG  -I'/home/ripley/R/Lib32/Rcpp/include' -I'/home/ripley/R/Lib32/RcppArmadillo/include' -I/opt/csw/include -I/usr/local/include  -fPIC  -O2  -c RcppExports.cpp -o RcppExports.o
/opt/csw/bin/g++ -std=gnu++11 -I"/home/ripley/R/gcc/include" -DNDEBUG  -I'/home/ripley/R/Lib32/Rcpp/include' -I'/home/ripley/R/Lib32/RcppArmadillo/include' -I/opt/csw/include -I/usr/local/include  -fPIC  -O2  -c utils.cpp -o utils.o
utils.cpp: In function ‘double dec(Rcpp::IntegerVector, int)’:
utils.cpp:13:25: error: call of overloaded ‘pow(int, int)’ is ambiguous
     s += pow(2, p - I(i));
                         ^
In file included from /usr/include/math.h:15:0,
                 from /opt/csw/include/c++/5.2.0/cmath:44,
                 from /home/ripley/R/Lib32/Rcpp/include/Rcpp/platform/compiler.h:100,
                 from /home/ripley/R/Lib32/Rcpp/include/Rcpp/r/headers.h:67,
                 from /home/ripley/R/Lib32/Rcpp/include/RcppCommon.h:29,
                 from /home/ripley/R/Lib32/RcppArmadillo/include/RcppArmadilloForward.h:26,
                 from /home/ripley/R/Lib32/RcppArmadillo/include/RcppArmadillo.h:31,
                 from utils.cpp:1:
/opt/csw/lib/gcc/i386-pc-solaris2.10/5.2.0/include-fixed/iso/math_iso.h:204:21: note: candidate: long double std::pow(long double, long double)
  inline long double pow(long double __X, long double __Y) { return
                     ^
/opt/csw/lib/gcc/i386-pc-solaris2.10/5.2.0/include-fixed/iso/math_iso.h:171:15: note: candidate: float std::pow(float, float)
  inline float pow(float __X, float __Y) { return __powf(__X, __Y); }
               ^
/opt/csw/lib/gcc/i386-pc-solaris2.10/5.2.0/include-fixed/iso/math_iso.h:72:15: note: candidate: double std::pow(double, double)
 extern double pow __P((double, double));
               ^
*** Error code 1
make: Fatal error: Command failed for target `utils.o'
Current working directory /tmp/Rtmpz9aisu/R.INSTALL289139384174/dga/src
ERROR: compilation failed for package ‘dga’
* removing ‘/home/ripley/R/Lib32/dga’
```

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -2,7 +2,7 @@ Package: dga
 Type: Package
 Title: Capture-Recapture Estimation using Bayesian Model Averaging
 Version: 2.0.1
-Date: 2021-05-03
+Date: 2021-05-04
 Authors@R: c(person(given="James", family="Johndrow", role=c("aut")),
              person(given="Kristian", family="Lum", email="kl@hrdag.org", role=c("aut")),
              person(given="Patrick", family="Ball", role=c("aut")),

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -1,7 +1,7 @@
 Package: dga
 Type: Package
 Title: Capture-Recapture Estimation using Bayesian Model Averaging
-Version: 2.0.0
+Version: 2.0.1
 Date: 2021-05-03
 Authors@R: c(person(given="James", family="Johndrow", role=c("aut")),
              person(given="Kristian", family="Lum", email="kl@hrdag.org", role=c("aut")),

--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -10,7 +10,7 @@ arma::rowvec colSumsSub(const arma::mat& mat,
 double dec(const Rcpp::IntegerVector I, int p) {
   double s = 0;
   for (int i = 0; i < I.size(); i++) {
-    s += pow(2, p - I(i));
+    s += pow(2.0, p - I(i));
   }
   return s;
 }

